app/services/commands/notification/create.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Commands
```

Comment:
I've been writing Commands for ServiceDays and attendances because they're complex and have lots of callback behavior I'd like to call explicitly.  I don't think it's necessary to have a create command for notifications.